packages/plugins/content/image/src/Component/index.js

```diff
@@ -25,15 +25,14 @@ import React from 'react'
 import Display from './Display'
 import Form from './Form'
 import type { ContentPluginProps } from 'ory-editor-core/lib/service/plugin/classes'
-import getMuiTheme from 'material-ui/styles/getMuiTheme'
-import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
+import ThemeProvider from 'ory-editor-ui/lib/ThemeProvider'
```

Comment:
The idea here is to have a central ThemeProvider and only set the theme up once?

Reply:
The main reason to have a central theme provider is that this gives us a way to share generateClassName and avoid generating classes with conflicting names.

Reply:
I see, that makes sense!